src/utils/promptUtils.c

```diff
@@ -37,7 +40,13 @@ char* getEncryptionPasswordFor(const char* forWhat,
                                const char* pw_cmd, const char* pw_file,
                                const char* pw_env) {
   if (pw_env != NULL) {
+	#ifdef __MSYS__
+    char value_buffer[256];
+    getRegistryEntry(pw_env, value_buffer);
+    char* pass = oidc_strcopy(value_buffer);
```

Comment:
After copying the value, it would be nice to remove the maybe sensitive content from the `value_buffer`. (In this case it is a password so it is sensitive, but please add this for all of the calls of `getRegistryEntry`.
The `value_buffer` should be cleared with 
```c
moresecure_memzero(value_buffer, 256);
```

Reply:
You actually could add a function to `utils/registryConnector.c`:

```c
char* getRegistryValue(const char* env_name) {
    char value_buffer[256];
    getRegistryEntry(env_name, value_buffer);
    char* value = oidc_strcopy(value_buffer);
    moresecure_memzero(value_buffer, sizeof(value_buffer));
    return value;
}
```